src/components/Missions/MissionsTable.js

```diff
@@ -0,0 +1,34 @@
+import { React } from 'react';
+import { useSelector } from 'react-redux';
+import MissionsInfo from './MissionsInfo';
+import './Missions.css';
+
+const MissionsTable = () => {
+  const missions = useSelector((state) => state.missions);
```

Comment:
- [OPTIONAL] It would have been better to pass `missions` as a `prop` to the `MissionsTable` component. This is because `missions` are again selected in the `Missions` component where this very component is rendered. This way, we are keeping our code DRY.